pulsar-broker/src/test/java/org/apache/pulsar/broker/intercept/InterceptFilterOutTest.java

```diff
@@ -71,7 +71,7 @@ public void testFilterOutForPreInterceptFilter() throws Exception {
         Mockito.doNothing().when(chain).doFilter(Mockito.any(), Mockito.any());
         HttpServletRequestWrapper mockInputStream = new MockRequestWrapper(request);
         Mockito.doReturn(mockInputStream.getInputStream()).when(request).getInputStream();
-        Mockito.doReturn(new StringBuffer("http://127.0.0.1:8080")).when(request).getRequestURL();
+        Mockito.doReturn(new StringBuilder("http://127.0.0.1:8080")).when(request).getRequestURL();
```

Comment:
method signature is   `StringBuffer getRequestURL();`, so can't use StringBuilder here.

Reply:
yes, I saw the failed case.
I plan to exclude this rule.

I created an issue:
https://github.com/gaul/modernizer-maven-plugin/issues/149
they insist this as a default rule.